api/controllers/EAppSubmittedController.js

```diff
@@ -37,8 +37,45 @@ const EAppSubmittedController = {
         }
     },
 
-    _renderPage(res) {
-        return res.view('eApostilles/applicationSubmissionSuccessful.ejs', {});
+    _renderPageAndSendConfirmationEmail(req, res) {
+        const queryParams = req.params.all();
+        const applicationId = queryParams.merchantReference;
+        const userDetails = {
+            firstName: req.session.account.first_name,
+            lastName: req.session.account.last_name,
+            email: req.session.email,
+            appType: req.session.appType,
+            userRef: req.session.user.id,
+        };
+
+        EAppSubmittedController._sendConfirmationEmail(
```

Comment:
minor - but is there a reason why `EAppSubmittedController.method` is used instead of `this.method`?

Reply:
I guess it's because I haven't instantiated the object so by default all the methods are static (if that makes sense). If I was using a class or a function constructor I think the `this` keyword would make sense.

Reply:
does sails do anything special with the controllers? it's probably not worth doing now since it might look odd when compared to the rest of the codebase, but you could unnest all of the methods and export all of the ones that are public, and not export the private ones. 

your file would look something like this
```.js 
// EAppSubmittedController.js
export function async addDocsAndRenderPage(req, res) {...}

function renderPageAndSendConfirmationEmail(req, res) {...}

// anotherFileThatUsesEAppSubmittedController.js
import { * as EAppSubmittedController } from '../somewhere/EAppSubmittedController'  // imports all of the functions that are "public" 

// so you can use it like so:
EAppSubmittedController.addDocsAndRenderPage(..)

// but not this since it's not exported 
EAppSubmittedController.renderPageAndSendConfirmationEmail(..)

```